docs/manual/layout/templates/twig/manage.en.md

```diff
@@ -0,0 +1,121 @@
+---
+title: 'Manage templates'
+description: 'The management of the template files.'
+aliases:
+    - /en/layout/templates/twig/manage-template/
+weight: 20
+---
+
+
+Twig templates are marked differently compared to the previous PHP templates: (`.html.twig` instead of `.html5`).
+
+{{% notice tip %}}
+Contao 5 includes ready-made Twig templates and these are favored. However, you can also activate and use the corresponding PHP-templates. 
+More information can be found [here](https://github.com/contao/contao/blob/5.x/UPGRADE.md#content-elements).
+{{% /notice %}}
+
+## Create Template
+
+{{< tabs groupId="creation">}}
+
+{{% tab name="Ab Contao 4.13.10" %}}
+
+Ready-made Twig Temlates are not available in this version and cannot be created via the backend. You have to create a file manually 
+in the `templates` directory, e.g. `ce_text.html.twig`. This Twig template will then be displayed in the backend 
+(could be copied and renamed here) and is then available for selection in the respective content element.
+
+It is possible to extend existing PHP-template from within Twig templates. For example, create a `fe_page.html.twig` in your 
+template directory. In this example, a heading is added above the main section and everything else stays the same:
```

Comment:
I don't think it's most important to denote which templates the core is shipping (this could be added in an info box for instance) but *that* you can overwrite/extend any template:

```suggestion
You can now use Twig templates to overwrite or extend PHP templates. Keep the file name like before, but adjust the file extension. For example, create a `fe_page.html.twig` in your `template` directory and add another heading above the main section:
```

Reply:
Done